pages/common/2to3.md

```diff
@@ -1,28 +1,32 @@
 # 2to3
 
-> Automated Python 2 to 3 code translation.
+> Automated Python 2 to 3 code conversion.
 > More information: <https://docs.python.org/3/library/2to3.html>.
 
-- Display changes that would be performed:
-
-`2to3 {{path/to/file.py}}`
-
 - Convert a Python 2 file to Python 3:
 
 `2to3 --write {{path/to/file.py}}`
 
-- Convert a specified Python 2 feature to Python 3:
+- Display the changes that would be performed without performing them (dry-run):
+
+`2to3 {{path/to/file.py}}`
```

Comment:
I'd suggest this example belongs at the beginning as it was before.

Reply:
Ah yes, I agree.  I think I used it because I think it's more common, but it's better to keep it at the top.